vs/dosbox.vcxproj.filters

```diff
@@ -1,814 +1,823 @@
+      <UniqueIdentifier>{075d87f5-0708-414a-a8c0-4284e1744cb0}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\hardware\mame">
-       <UniqueIdentifier>{375a1299-bfa1-4e28-ae0f-7769b8b1653b}</UniqueIdentifier>
+      <UniqueIdentifier>{375a1299-bfa1-4e28-ae0f-7769b8b1653b}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\hardware\serialport">
-       <UniqueIdentifier>{5130c0ee-83d9-4e35-b61e-fb189d1e1b31}</UniqueIdentifier>
+      <UniqueIdentifier>{5130c0ee-83d9-4e35-b61e-fb189d1e1b31}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\ints">
-       <UniqueIdentifier>{35d85ae9-4456-4739-b10b-92b567bcf470}</UniqueIdentifier>
+      <UniqueIdentifier>{35d85ae9-4456-4739-b10b-92b567bcf470}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\libs">
-       <UniqueIdentifier>{0dd0e2e6-f72d-4847-bc5a-bf41bcea4171}</UniqueIdentifier>
+      <UniqueIdentifier>{0dd0e2e6-f72d-4847-bc5a-bf41bcea4171}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\libs\decoders">
-       <UniqueIdentifier>{d49f3a50-e4ed-4da7-a530-eeefef580e9b}</UniqueIdentifier>
+      <UniqueIdentifier>{d49f3a50-e4ed-4da7-a530-eeefef580e9b}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\libs\gui_tk">
-       <UniqueIdentifier>{c1d4aca1-2ddd-4dc2-ba93-8b2279ef4d30}</UniqueIdentifier>
+      <UniqueIdentifier>{c1d4aca1-2ddd-4dc2-ba93-8b2279ef4d30}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\libs\nuked">
-       <UniqueIdentifier>{69d13cb3-0db7-4e3c-92a0-c2b1a79b5b63}</UniqueIdentifier>
+      <UniqueIdentifier>{69d13cb3-0db7-4e3c-92a0-c2b1a79b5b63}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\libs\ppscale">
-       <UniqueIdentifier>{63f12ee0-e983-404f-bbd2-5b6cd633a0af}</UniqueIdentifier>
+      <UniqueIdentifier>{63f12ee0-e983-404f-bbd2-5b6cd633a0af}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\misc">
-       <UniqueIdentifier>{cf5cb904-4d22-4482-8dfa-ab37306b91c8}</UniqueIdentifier>
+      <UniqueIdentifier>{cf5cb904-4d22-4482-8dfa-ab37306b91c8}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\platform">
-       <UniqueIdentifier>{cc5fe9d0-d137-4224-8849-81bb74933a1f}</UniqueIdentifier>
+      <UniqueIdentifier>{cc5fe9d0-d137-4224-8849-81bb74933a1f}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\platform\visualc">
-       <UniqueIdentifier>{f21cdf18-85a2-4afc-9bbd-aa6423b893de}</UniqueIdentifier>
+      <UniqueIdentifier>{f21cdf18-85a2-4afc-9bbd-aa6423b893de}</UniqueIdentifier>
     </Filter>
     <Filter Include="src\shell">
-       <UniqueIdentifier>{d7f601fb-7919-4adc-8a23-cc5c8a14ff94}</UniqueIdentifier>
+      <UniqueIdentifier>{d7f601fb-7919-4adc-8a23-cc5c8a14ff94}</UniqueIdentifier>
+    </Filter>
+    <Filter Include="src\midi">
+      <UniqueIdentifier>{5920d562-013f-417e-8095-198288c97b52}</UniqueIdentifier>
     </Filter>
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
(maybe missing a CR?)

Reply:
I updated Visual Studio, perhaps it now decided to no longer use newline for its XML files…